webapp/src/components/call_widget/component.scss

```diff
@@ -135,6 +135,8 @@ a.calls-channel-link {
 
 #calls-widget .Menu {
   position: static;
+  border: none;
+  box-shadow: none;
```

Comment:
Slightly unrelated but this was needed to fix some more issues when switching to Boards.